uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1022.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1022 {
+  
+  /**
+   * Hide Fields For Contact Summary
+   */
+  public function upgrade_1022() {
```

Comment:
Just a general question, if this upgrader is run twice will the result be the same? That's what we aim for with upgraders. I think this one looks fine, but just asking

Reply:
Great, thanks a lot for checking